tests/python/gaia-ui-tests/gaiatest/tests/functional/bzlite/test_search_bug.py

```diff
@@ -0,0 +1,13 @@
+from gaiatest import GaiaTestCase
+from gaiatest.apps.bzlite.app import BugzillaLite
+
+class TestSearchBug(GaiaTestCase):
+	def setUp(self):
+		GaiaTestCase.setUp(self)
+		self.connect_to_local_area_network()
+
+	def test_search_bug(self):
+		bugzilla_lite = BugzillaLite(self.marionette)
+		bugzilla_lite.launch()
+		bugzilla_lite.search(str(self.testvars['bugs']['5']))
+		bugzilla_lite.cancelSearch()
```

Comment:
I didn't managed to get to cancelSearch. I always failed at the line just before, where "No Bugs :(" was displayed.